modules/demo/src/main/scala/lucuma/ui/demo/Demo.scala

```diff
@@ -161,6 +185,21 @@ object FormComponent {
               changeAuditor = ChangeAuditor.bigDecimal(4),
               onValidChange = v => $.setStateL(State.bigDecimal)(v)
             ),
+            FormInputEV(
+              id = "refined-big-decimal",
+              label = "Refined Big Decimal - 1 decimal place",
+              value = $.props.root
+                .zoom(RootModel.refinedBigDec)
+                .zoomSplitEpi(
```

Comment:
Having number of decimals as a type parameter ensures that the number of decimals matches in this zoom and in the validFormat below.